### PR TITLE
don't use the admin to set the webservices user/pass in cluster doc

### LINKED ISSUE #2458: clustering doc: webservices configuration
At step 3.5.1 of the `PacketFence Clustering Guide`, it ask to use the administration GUI to set up a webservices username / password combination. At that point of configuration, database is running in read-only mode which makes the administration GUI configuration saving unavailable.

Options:
- Have the ability to save that one and only configuration parameter even in read-only mode;
- Modifiy documentation to set the webservices username / password combination manually in pf.conf and reload the configurations;

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
don't use the admin to set the webservices user/pass in cluster doc

# Impacts
Documentation for cluster setup

# Issue
fixes #2458 

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 1]
```diff
@@ -295,14 +295,21 @@ Now, you will need to integrate your 2 other nodes in your cluster.
...
-NOTE: If you access https://PF_HOSTNAME:1443, you will still be redirected to the configurator. In order to gain access to the full administration interface, open https://PF_HOSTNAME:1443/admin
+On the first server, configure your webservices username and password by adding the following in `pf.conf`:
 
-First, set the webservices user and password on the master node in 'Configuration/Integration/Web Services'. There should be a sync error you can safely ignore as the other servers of your cluster aren't joined yet.
+  [webservices]
+  user=packet
+  pass=fence
 
-And restart httpd.webservices on the first server:
+You can set the username and password to any value you wish, make sure you keep closeby for usage below.
```
Comment (louismunro) [inline docs/PacketFence_Clustering_Guide.asciidoc]: Suggestion:  While you can set the username and password to any value, make sure to keep it safe as you will need it while initializing the cluster below.

**Review by louismunro — APPROVED** [review]
Approved with minor comments.

**Review by dwlfrth — APPROVED** [review]
As @louismunro said

**Comment by julsemaan** [comment]
Should be all good now, used your exact sentence

**Pull request merged**